AIFG-3D-Connect-4/FourTechAI.cpp

```diff
@@ -43,4 +43,10 @@ Coordinate FourTechAI::getMove()
 	return bestMove->first; // Returns the position of the best move.
 }
 
+Coordinate FourTechAI::getCoordinate()
+{
+	// Will get the best move the AI wants.
+	return getMove();
```

Comment:
I'd just move the contents of getMove() this function/rename getMove() to getCoordinate(), no need to call it from within.